live/root/usr/lib/dracut/modules.d/99agama-cmdline/agama-network-compat.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+[ -e /dracut-state.sh ] && . /dracut-state.sh
+
+. /lib/dracut-lib.sh
+. /lib/net-lib.sh
+
+ifcfg_to_ip() {
+  local ip
+  local v="${2}",
+  local interface="$1"
+  local conf_path="/etc/cmdline.d/40-agama-network.conf"
+  set --
+  while [ -n "$v" ]; do
+    set -- "$@" "${v%%,*}"
+    v=${v#*,}
+  done
+
+  if [[ $# -eq 0 ]]; then
+    echo "IFCFG 0 options given, must be wrong"
+    return 1
+  fi
+
+  ### See https://en.opensuse.org/SDB:Linuxrc#Network_Config
+  # ifcfg=<interface_spec>=[try,]dhcp*,[rfc2132,]OPTION1=value1,OPTION2=value2...
+  if str_starts "$1" "dhcp"; then
+    autoconf="$1"
+    if [ "$autoconf" = "dhcp4" ]; then
+      echo "AUTOCONF"
+      autoconf="dhcp"
+    fi
+    case $autoconf in
+    "dhcp" | "dhcp6")
+      if [ "$interface" = "*" ]; then
+        echo "ip=${1}" >>"/etc/cmdline.d/40-agama-network.conf"
```

Comment:
why not $conf_path here? it should be same, not?

Reply:
should be there ... probably missing when modified...